inst/metadata/colour_scales.csv

```diff
@@ -1,5 +1,5 @@
 iso3,indicator,colour,min,max,invert_scale
-default,current_art,interpolateViridis,0,80000,TRUE
+default,art_current,interpolateViridis,0,80000,TRUE
 default,prevalence,interpolateMagma,0,0.5,TRUE
 default,art_coverage,interpolateViridis,0,1,FALSE
 default,vls,interpolateWarm,0,1,FALSE
```

Comment:
The references in this file to `vls` and `recent` need to be updated to `viral_suppression_plhiv` and `recent_infected`

Reply:
Thanks very much for catching these. Addressed in 8a5c7f3.